installation/environment-variables.md

```diff
@@ -57,18 +57,65 @@ description: >-
   </tbody>
 </table>
 
-## Examples - Linux
+## Examples - Kubernetes \(yml configuration\)
 
-```text
-ED_API_KEY=<your api key> \
-HTTPS_PROXY=<your proxy details> \
-bash -c "$(curl -L https://release.edgedelta.com/release/install.sh)"
+Snippet pulled from: [https://edgedelta.github.io/k8s/edgedelta-agent.yml](https://edgedelta.github.io/k8s/edgedelta-agent.yml)
+
+```yaml
+spec:
+  containers:
+  - name: edgedelta
+    image: docker.io/edgedelta/agent:latest
+    env:
+      - name: ED_API_KEY
+        valueFrom:
+          secretKeyRef:
+            key: ed-api-key
+            name: ed-api-key
+      - name: HTTPS_PROXY
+        value: <your proxy details>
 
 ```
+### Examples - Kubernetes Secret \(yml configuration\)
```

Comment:
the above kubernetes example also has secreto from environment. didn't understand the difference between these two versions of config. can we merge them and have just k8s example?

Reply:
Merged them.